eureka/S5_lightcurve_fitting/models/GPModel.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
+import george
```

Comment:
If george isn't already in the requirements.txt file, you should add it. Same for tinygp when you get around to testing that one out